conf/gunicorn.conf.py

```diff
@@ -0,0 +1,2 @@
+bind = "0.0.0.0:8001"
+worker_tmp_dir = "/dev/shm"
```

Comment:
`worker_tmp_dir` came from https://github.com/hypothesis/lms/pull/4169. I guess we may want to set this in production for all our apps @marcospri?